snap/snapcraft.yaml

```diff
@@ -78,65 +88,111 @@ parts:
     override-build: |
       snapcraftctl build
       sed -i 's|Icon=dolphin-emu|Icon=/usr/local/share/icons/hicolor/scalable/apps/dolphin-emu.svg|' $SNAPCRAFT_PART_INSTALL/usr/local/share/applications/dolphin-emu.desktop
+      sed -i 's|Exec=env QT_QPA_PLATFORM=xcb dolphin-emu|Exec=dolphin-emu|' $SNAPCRAFT_PART_INSTALL/usr/local/share/applications/dolphin-emu.desktop
+    organize: 
+        #Copy Sys folder to the Binaries folder
+        usr/local/share/dolphin-emu/sys: usr/local/bin/Sys
     after:
       - patches
-  desktop-qt5:
-    build-packages:
-      - build-essential
-      - qtbase5-dev
-      - dpkg-dev
-    make-parameters:
-      - FLAVOR=qt5
-    plugin: make
-    source: https://github.com/ubuntu/snapcraft-desktop-helpers.git
-    source-subdir: qt
-    stage-packages:
-      - libxkbcommon0
-      - ttf-ubuntu-font-family
-      - dmz-cursor-theme
-      - light-themes
-      - adwaita-icon-theme
-      - gnome-themes-standard
-      - shared-mime-info
-      - libqt5gui5
-      - libgdk-pixbuf2.0-0
-      - libqt5svg5
-      - try:
-          - appmenu-qt5
-      - locales-all
-      - xdg-user-dirs
-      - libgtk2.0-0
-      - qt5-gtk-platformtheme
-      # Are all the below needed?
-      # - libqt5multimedia5-plugins
-      - qtmultimedia5-dev
-      - qtwayland5
-    override-prime: |
-      snapcraftctl prime
-      sed -i 's|XDG_DATA_HOME=$SNAP_USER_DATA|XDG_DATA_HOME=$SNAP_USER_COMMON|' $SNAPCRAFT_PRIME/bin/desktop-launch
-      sed -i 's|XDG_CONFIG_HOME=$SNAP_USER_DATA|XDG_CONFIG_HOME=$SNAP_USER_COMMON|' $SNAPCRAFT_PRIME/bin/desktop-launch
+  # desktop-qt5:
+  #   build-packages:
+  #     - build-essential
+  #     - qtbase5-dev
+  #     - dpkg-dev
+  #   make-parameters:
+  #     - FLAVOR=qt5
+  #   plugin: make
+  #   source: https://github.com/ubuntu/snapcraft-desktop-helpers.git
+  #   source-subdir: qt
+  #   stage-packages:
+  #     - libxkbcommon0
+  #     - ttf-ubuntu-font-family
+  #     - dmz-cursor-theme
+  #     - light-themes
+  #     - adwaita-icon-theme
+  #     - gnome-themes-standard
+  #     - shared-mime-info
+  #     - libqt5gui5
+  #     - libgdk-pixbuf2.0-0
+  #     - libqt5svg5
+  #     - try:
+  #         - appmenu-qt5
+  #     - locales-all
+  #     # - xdg-user-dirs
+  #     - libgtk2.0-0
+  #     - xdg-utils
+  #     - qt5-gtk-platformtheme
+  #   #   # Are all the below needed?
+  #   #   # - libqt5multimedia5-plugins
+  #   #   - qtmultimedia5-dev
+  #     - qtwayland5
+  #   override-prime: |
+  #     snapcraftctl prime
+  #     sed -i 's|XDG_DATA_HOME=$SNAP_USER_DATA|XDG_DATA_HOME=$SNAP_USER_COMMON|' $SNAPCRAFT_PRIME/bin/desktop-launch
+  #     sed -i 's|XDG_CONFIG_HOME=$SNAP_USER_DATA|XDG_CONFIG_HOME=$SNAP_USER_COMMON|' $SNAPCRAFT_PRIME/bin/desktop-launch
+  #     sed -i '2iset -x' $SNAPCRAFT_PRIME/bin/desktop-launch
+
```

Comment:
Good to remove `desktop-qt5`; this is incompatible with the kde-frameworks content snaps.